src/Test/Perf/util/test_util.csx

```diff
@@ -17,33 +17,36 @@ using System.Net;
 // Arguments
 //
 
+// This is due to a "bug" in csi that has Args non-static.  Thus, you can't 
+// use it from non-static methods.
```

Comment:
I see...hence the ```"..."```...

FWIW, I still find this cumbersome.  It would be better if I didn't have to mark the TLDs ```static```.